src/main/index.js

```diff
@@ -91,14 +80,6 @@ const menu = [
   {
     label: 'Locations',
     submenu: [
-      {
```

Comment:
I would like to remove this and have a better option (like a popup window) for user to add new locations instead in the future.

Besides editing a config file sounds tacky. If the user edits the config file, assuming the user knows what to do, the menu will not update with the new location (or deleted locations) anyway.

Also I think I had issues maintaining location file when I turned the application to a Desktop App (gotta check if it can compile to an app all the time, right?)

Reply:
Also can you check if it runs on Linux? Just go to `src/renderer/index.js` and try to import `serialport` into the code. If serialport doesnt crash the whole thing it should work. No need to test with specific Xbee ports since I have done that. As long as you know the specific port name for Linux you should be fine. Like Windows has the `COM5` and macOS has the `/dev/tty.portETC`, not sure for Linux

Reply:
I don't have an xbee on me or at home. Does that matter?

Reply:
No. Just see if the program runs with `import 'serialport';` in it